collection-scripts/gather_vms_details

```diff
@@ -100,6 +100,8 @@ export -f get_vm_rule_tables
 
 "${DIR_NAME}"/vmConvertor
 
+"${DIR_NAME}"/gather_ns
```

Comment:
@machadovilaca - what do you think about adding a parameter to this script (`gather_ns`), to only gather ns info from the installation ns?

Reply:
discussed this possibility with @tiraboschi and we think that collecting the info from all related namespaces won't be problematic, even with large clusters